docs/multi-cluster-install.md

```diff
@@ -71,7 +72,8 @@ Run the following commands
 Setup the environment with your specific values.
 
 ```shell
-API_SERVER_URL="https://example.com:6443"
+# replace CLUSTERNAME with the name of the cluster according to your ${HOME}/.kube/config
+API_SERVER_URL=$(kubectl config view -o json --raw  | jq -r '.clusters[] | select(.name=="CLUSTER").cluster["server"]')
```

Comment:
Same, I would recommend two section, one for single cluster and one for multi-cluster.